tools/build/setnextversion.sh

```diff
@@ -131,7 +131,6 @@ echo "setting new version numbers"
 mvn versions:set -DnewVersion=$version -P vmware -P developer -P systemvm -P simulator -Dnoredist versions:commit
 
 perl -pi -e "s/$currentversion/$version/" debian/changelog
-perl -pi -e "s/$currentversion/$version/" tools/apidoc/pom.xml
```

Comment:
@weizhouapache there's no apidoc pom.xml? or data in it to sed/grep (btw sed would be faster than perl, but this is good too)

Reply:
@rohityadavcloud 
the version number in tools/apidoc/pom.xml has already been updated by line 131
```
mvn versions:set -DnewVersion=$version -P vmware -P developer -P systemvm -P simulator -Dnoredist versions:commit
```

This fixes the issue that debian packages cannot be build in private repository.